data/ships.txt

```diff
@@ -178,6 +178,109 @@ ship "Arrow"
 
 
 
+ship "Auxiliary"
+	sprite "ship/auxiliaryh"
+	thumbnail "thumbnail/auxiliaryh"
+	attributes
+		category "Transport"
+		licenses
+			Navy
+			Auxiliary
+		"cost" 13720000
+		"shields" 17100
+		"hull" 5900
+		"required crew" 87
+		"bunks" 260
+		"mass" 820
+		"drag" 13.3
+		"heat dissipation" .45
+		"fuel capacity" 900
+		"cargo space" 250
+		"outfit space" 720
+		"weapon capacity" 160
+		"engine capacity" 200
+		weapon
+			"blast radius" 280
+			"shield damage" 2800
+			"hull damage" 1400
+			"hit force" 4200
+	outfits
+		"Sidewinder Missile Launcher" 2
+		"Sidewinder Missile" 100
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret" 2
+		
+		"Fusion Reactor"
+		"LP288a Battery Pack"
+		"D94-YV Shield Generator"
+		"D67-TM Shield Generator"
+		"Water Coolant System"
+		"Large Radar Jammer"
+		"Ramscoop"
+		
+		"X4700 Ion Thruster"
+		"X1700 Ion Thruster"
+		"X5200 Ion Steering"
+		"Scram Drive"
+	engine -41 181
+	engine 41 181
+	gun -12 -180
+	gun 12 -180
+	turret 0 -75 "Heavy Anti-Missile Turret"
+	turret 0 -18
+	turret 0 22.5
+	turret 0 73.5 "Heavy Anti-Missile Turret"
+	fighter 0 -75
+	fighter 0 -18
+	fighter 0 22.5
+	fighter 0 73.5
+	leak "leak" 30 50
+	leak "flame" 30 80
+	leak "big leak" 30 30
+	explode "tiny explosion" 20
+	explode "small explosion" 45
+	explode "medium explosion" 50
+	explode "large explosion" 40
+	explode "huge explosion" 10
+	"final explode" "final explosion large"
+	description "The Auxiliary acts as the logistical backbone of the Navy. By being built in stages the Auxiliary has the ability to act as a transport, freighter, or a hybrid of the two."
```

Comment:
How about this instead?

"The Auxiliary is as the Republic Navy's logistical backbone. Built using flexible modules, the Auxiliary can serve as a transport, freighter, or a hybrid of the two."

Reply:
Yeah, that sounds much better